tokens.json

```diff
@@ -1153,5 +1153,15 @@
       "twitter": "https://twitter.com/KryptoThreadz",
       "discord": "https://discord.gg/2Fs4ERC9Re"
     }
+  },
+  "9abf0afd2f236a19f2842d502d0450cbcd9c79f123a9708f96fd9b96": {
+    "project": "ENCOINS $ENCS",
+    "categories": ["DeFi", "DAO", "Other"]
```

Comment:
Your PR is failed CI. You missed a comma here

Reply:
Thanks, fixed.